cmd/generate.go

```diff
@@ -18,17 +20,19 @@ const (
 )
 
 type generateCmd struct {
-	apimodelPath      string
-	outputDirectory   string // can be auto-determined from clusterDefinition
-	caCertificatePath string
-	caPrivateKeyPath  string
-	classicMode       bool
-	noPrettyPrint     bool
-	parametersOnly    bool
+	translationsDirectory string
```

Comment:
You are right, localization of ACS Engine command has lower priority. The primary translation happens in acsengine and api package, for the purpose of RP. I added here for the e2e test and it is nice to have for ACS Engine. For the translationsDirectory  parameter, it will not be needed and default to the translation folder if acs-engine is run at the package root directory (I think that's the most used scenario?) It has the issue if go install is used without deploy translation file. Another approach is to used go-bindata to avoid such caveat. I'll look into it first.